labs/fabric-netkit.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Fabric Netkit
+parent: Labs
+---
+
+# Lab Name
+
+[Fabric Netkit](https://github.com/hyperledger-labs/fabric-netkit)
+
+# Short Description
+
+Software development kit for access and management of Hyperledger Fabric network
+
+# Scope of Lab
+
+The scope of this lab is to deliver the software development kit for access and management Hyperledger Fabric network:
+
+* query and invoke chaincodes, using local configuration and service discovery
+* sdk for interacting with system chaincodes (csss, lscc, qscc, lifecycle)
+* deliver and parse blocks
+* build, install and instantiate chaincodes
+* configure channels
+
+# Initial Committers
+
+- https://github.com/vitiko
+- https://github.com/dimaxgl
+- https://github.com/bogatyr285
+- https://github.com/dimuls
+- https://github.com/inotnako
+- https://github.com/criro1
+
+# Sponsor
+
+- https://github.com/C0rWin - HLF maintainer, Hyperledger TSC Member
+
+# Pre-existing repository
+
+- https://github.com/s7techlab/hlf-sdk-go
```

Comment:
How does this compare to the existing fabric-sdk-go repo? What are the differences/similarities? Have you talked with the maintainers of that repo to add the pieces that are missing?

Reply:
I agree with @tkuhrt It's important to explain the motivation for trying to develop a different SDK. The README in the existing repo doesn't really provide any for that matter.
Note that this should really be compared against the new fabric-gateway SDK, which provides for a higher level API, given that the old Golang SDK is being retired.

Reply:
It will be continue of work, started in existing fabric-sdk-go repo. Maintainers ( @vitiko and @dimaxgl) prefer to work on this project under the hyperledger-labs namespace. Differences - project will be enriched with features, related with chaincode packaging, peer and orderer management.

Reply:
@lehors , this project aim to be not only sdk, but kit for managing HLF network - building chaincodes, installing and commiting chaincodes to peer, configuring channels.

Reply:
> I agree with @tkuhrt It's important to explain the motivation for trying to develop a different SDK. The README in the existing repo doesn't really provide any for that matter. Note that this should really be compared against the new fabric-gateway SDK, which provides for a higher level API, given that the old Golang SDK is being retired.

The key difference is more flexible programming model, the standard SDK is rather opionated on the way it enables you, for example, to cope with signing identity. Or to issue commands to operate with Fabric network, as @vitiko mentioned Fabric-NetKit has also a programming model to operate with Fabric network Facilities exposing convenient API for the client side.